practice-two/src/js/script.js

```diff
@@ -150,24 +158,42 @@ const isvalidateForm = () => {
  * 
  */
 const saveData = () => {
-  // Parse any JSON previously stored in allEntries
-  let existingEntries = JSON.parse(localStorage.getItem(UsersKey));
+  // Append values ​​to array 
+  existingEntries.push({
+    fullName: fullNameInput.value,
+    email: emailInput.value,
+    salary: salaryInput.value,
+    city: cityInput.value
+  });
 
-  // If existingEntries is null, array will be created
-  if(existingEntries === null) {
-    existingEntries = [];
-  } else {
-    // Append values ​​to array 
-    existingEntries.push({
-      fullName: fullNameInput.value,
-      email: emailInput.value,
-      salary: salaryInput.value,
-      city: cityInput.value
-    });
-  }
   localStorage.setItem(UsersKey, JSON.stringify(existingEntries));
 }
 
+/**
+ * Show user data in table
+ */
+const renderUserTable = () => {
+  let tableTemplate = '';
+  existingEntries.forEach(element => {
+    tableTemplate += ` 
+      <tbody> 
+        <tr>
+          <td>${element.fullName}</td>
+          <td>${element.email}</td>
+          <td>${element.salary}</td>
+          <td>${element.city}</td>
+          <td>
+            <button>Delete</button>
+          </td>
+        </tr>
+      </tbody>`;
+  });
+  
+  document.getElementById('userTableBody').innerHTML = tableTemplate;    
+}
+
+renderUserTable();
```

Comment:
For code that have to run when we run application the first time. It should be on top or bottom all, not in the middle